src/_posts/platform/app/2000-01-01-log-drain.md

```diff
@@ -78,6 +78,18 @@ Parameters:
 This variable is available on
 [AppSignal documentation](https://docs.appsignal.com/logging/configuration.html).
 
+#### Better Stack Logs
+```bash
+scalingo --app my-app log-drains-add --type logtail \
```

Comment:
Shouldn't we update the CLI to have the type `better-stack`?

Reply:
There is too many changes to do, i will explain it in the ticket

Reply:
For now i will merge it. We can change the name in the CLI afterwards as in the API